libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.cpp

```diff
@@ -1377,6 +1381,21 @@ bool WiFiClientSecure::loadPrivateKey(Stream& stream, size_t size) {
 // SSL debugging which should focus on the WiFiClientBearSSL objects.
 
 extern "C" {
+
+#if CORE_MOCK
+
+  void br_esp8266_stack_proxy_init(uint8_t *space, uint16_t size) {
+    (void)space;
+    (void)size;
+  }
+  void _BearSSLCheckStack(const char *fcn, const char *file, int line) {
+    (void)fcn;
+    (void)file;
+    (void)line;
+  }
+
+#else // !CORE_MOCK
+
   extern size_t br_esp8266_stack_proxy_usage();
```

Comment:
Ah, poo.  These 2 (BSSLChkStack, BSSLPrint) routines are no longer used with BearSSL as a thunk.  Would you like to save the effort and remove them here, or do you want me to do a separate PR?

Reply:
Because you are offering, I'd be glad to leave it to you

Reply:
Will do after this merges.  You've got enough commits already!